packages/manager/apps/telecom/src/app/telecom/telephony/alias/dashboard/dashboard.controller.js

```diff
@@ -59,7 +66,17 @@ export default class TelecomTelephonyAliasHomeController {
       ),
     };
 
-    this.fetchService();
+    this.fetchService().then(() => {
```

Comment:
I will add it, i don't know if it's necessary or not but it doesn't cost anything to add it and may be safer to avoid some weird behavior ! 

Thanks !